### DISCUSSION TITLE
new Instance up, how to log in as admin?

### DISCUSSION THREAD

**Discussion by FelixSteindorff** [mastodon/mastodon#20493] [General]
Hey guys,

I just set up my new instance at elest.io and now want to log in into my admin account to configure all settings and contents.
But how does that exactly work? I got my admin credentials displayed in the managing panel at elest.io, but can't log in with them.
Any help would be appreciated.
I have to add that I never worked with linux and Ubuntu before, so don't know if I have to connect via SSH to configure other stuff before.

**Comment by KrisEnnkay** [comment]
Have you tried logging in with your email address instead of your username?
